configuration/config2/Utils.cpp

```diff
@@ -0,0 +1,35 @@
+/*##############################################################################
```

Comment:
I put this code in a common module for use by config2 manager. It's a static function that is used by a number of methods. Being located in the config2 directory I figured was enough to indicate it's for config2. If this should be moved to a system wide location, please tell me where. I am hesitant to move it unless it becomes part of a system lib because it would only be compiled and used by config2.